docs/fides/docs/development/releases.md

```diff
@@ -193,12 +193,13 @@ Next, run the following checks via the test environment:
 
 Run these from within the test environment shell:
 
-* [ ] Make sure to login your CLI user by running `fides user login -u root_user -p Testpassword1!`
-* [ ] Run a `fides push`
-* [ ] Run a `fides pull`
-* [ ] Run a `fides evaluate`
-* [ ] Generate a dataset with `fides generate dataset db --credentials-id app_postgres test.yml`
-* [ ] Scan a database with `fides scan dataset db --credentials-id app_postgres`
+* [ ] `git reset --hard` - **Note: This is required for the `pull` command to work**
```

Comment:
This might be useful just above the fides pull line instead

Reply:
I figured it might just be safest to get it out of the way first, so that the rest are pure `fides` commands

Just my own subjective logic though 🙂